packages/astro/src/@types/astro.ts

```diff
@@ -429,6 +429,23 @@ export interface AstroUserConfig {
 	 */
 	outDir?: string;
 
+	/**
+	 * @docs
+	 * @name cacheDir
+	 * @type {string}
+	 * @default `"./node_modules/.astro"`
+	 * @description Set the directory used to cache build artifacts to. Files in this directory will be used in subsequent builds to speed up the build time.
```

Comment:
```suggestion
	 * @description Set the directory for caching build artifacts. Files in this directory will be used in subsequent builds to speed up the build time.
```
I think this is shorter and sweeter!